Sources/ArcGISToolkit/Components/Offline/OfflineMapViewModel.swift

```diff
@@ -112,6 +112,15 @@ class OfflineMapViewModel: ObservableObject {
         
         // Note: We don't reset the mode once it is determined.
         
+        // Before loading models, check if the Internet is available.
+        do {
+            _ = try await offlineMapTask.preplannedMapAreas
+        } catch {
+            if error.isNoInternetConnectionError {
+                return
+            }
+        }
```

Comment:
In order to check the internet connection, I need to make a network request and get the `URLError` to use `isNoInternetConnectionError` extension.

`onlineMap.retryLoad()` only make a network request when the current `loadStatus` isn't `loaded`, so it cannot be used here.

Reply:
This is no longer needed as the network connection is checked by `loadPreplannedMapModels()` method.